CHANGELOG.md

```diff
@@ -1,23 +1,34 @@
 # Unreleased
- 
+
+# 9.15.0 (Feature release)
+
 ## New features
 
-### Replace back link placeholder URLs with javascript
+### Replace back link placeholder URLs with JavaScript
+
+We've added JavaScript to make the [back link component](https://design-system.service.gov.uk/components/back-link/) take users to the previous page by default.
+
+Before, to get the back link to work, you had to manually update placeholder text.
```

Comment:
```suggestion
We've added JavaScript to make the [back link component](https://design-system.service.gov.uk/components/back-link/) take users to the previous page by default, rather than you having to update placeholder text.
```

Reply:
blending these 2 as the 'Before' line sounded a little clunky out of context